staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -322,49 +328,30 @@ func TestApfCancelWaitRequest(t *testing.T) {
 		t.Error(err)
 	}
 
-	checkForExpectedMetricsWithRetry(t, []string{
+	checkForExpectedMetrics(t, []string{
 		"apiserver_current_inflight_requests",
 		"apiserver_request_terminations_total",
 		"apiserver_dropped_requests_total",
 	})
 }
 
 // wait async metrics to be collected
```

Comment:
is this comment still correct?

Reply:
It was not really correct in the first place.  What was async was the requests that caused metric updates.

Reply:
updated go doc